lib/ret/media_search.ex

```diff
@@ -69,6 +69,10 @@ defmodule Ret.MediaSearch do
     favorites_search(cursor, type, account_id, q)
   end
 
+  def search(%Ret.MediaSearchQuery{source: "public_rooms", cursor: cursor, q: q}) do
```

Comment:
I'd refactor this API to be just `room` search, and for now its an implicit filter of only public rooms. (We could revisit this if/when we add more favorite types, and then could roll favorites into being a filter here and elsewhere, etc)

Reply:
Changed the source to "rooms" and added an explicit "public" filter. Doing it implicitly feels like it will be more of a pain to rollback later